docs/medical-api/api-reference/fhir/consolidated-data-query-post.mdx

```diff
@@ -42,14 +42,10 @@ that will contain a URL to retrieve the data from.
   supported resources.
 </Tip>
 
-<Info>
-  Some general resources, like `Practitioner` do not belong to a specific patient - when you query
-  for these, all resources of that type will be returned.
-</Info>
-
 <ParamField query="resources" type="string" optional>
   A comma separated, case sensitive list of resources to be returned. If none are provided all
-  resources will be included.
+  resources will be included. Metriport with automatically hydrate the initial filtered resources 
+  with referenced resources to create bundle consistency.
```

Comment:
nit: Metriport will automatically hydrate the initially filtered resources with referenced ones to create bundle consistency.